fe/fe-core/src/main/java/org/apache/doris/service/ExecuteEnv.java

```diff
@@ -42,11 +43,6 @@ public static ExecuteEnv getInstance() {
         return INSTANCE;
     }
 
-    public static void setup() {
-        ExecuteEnv env = getInstance();
-        env.scheduler = new ConnectScheduler(Config.qe_max_connection);
-    }
-
     public ConnectScheduler getScheduler() {
```

Comment:
If you do this, then the `getScheduler()` and `getMultiLoadMgr()` need to be changed too.

Reply:
I think this change will not affect these two methods. Can you explain it？

Reply:
Sorry for replying late.
I misunderstood. it looks good to me.